src/InputMoney.js

```diff
@@ -0,0 +1,25 @@
+const Utils = require("./Utils");
+const { ERROR_MESSAGES } = require("./Constant");
+
+class InputMoney {
+  constructor(money) {
+    this.utils = new Utils();
+    this.validate(money);
+  }
+
+  validate(money) {
+    (this.validateMoney(money) === false &&
+      this.utils.throwError(ERROR_MESSAGES.ERROR_MONEY));
```

Comment:
이 경우에는 `this.validateMoney(money) === false`를 `!this.validateMoney(money)`로 바꾸어 줄 수 있을 것 같아요!
하지만 저라면 이 문장은 if 문을 사용하거나 반환을 시켜줄 것 같아요..! `eslint: no-unused-expressions`에 걸리기도 하구요!

```suggestion
    if(!this.validateMoney(money)) {
      this.utils.throwError(ERROR_MESSAGES.ERROR_MONEY)); 
    }
    //  아니면
   return !this.validateMoney(money)) && this.utils.throwError(ERROR_MESSAGES.ERROR_MONEY));
```

https://eslint.org/docs/latest/rules/no-unused-expressions